gitlab/v4/objects.py

```diff
@@ -1940,6 +1942,12 @@ def create(self, data, **kwargs):
         return CreateMixin.create(self, data, path=path, **kwargs)
 
 
+class ProjectPipelineJobManager(RetrieveMixin, RESTManager):
```

Comment:
Could you use `GetWithoutIdMixin` instead of `RetrieveMixin`?

It looks like getting a single job is not possible with the GitLab API, but we can simulate this feature by looping through the list.